contracts/FantomAuction.sol

```diff
@@ -419,16 +419,6 @@ contract FantomAuction is
 
         // Ensure there is a winner
         require(_winner != address(0), "no open bids");
-        require(
-            _winningBid >= auction.reservePrice,
-            "highest bid is below reservePrice"
-        );
```

Comment:
This check has to stay in place for the winner! Winner must not be able to result underpriced auction!
You can change the condition to: `_winningBid >= auction.reservePrice || _msgSender() == seller`